package.json

```diff
@@ -2,6 +2,8 @@
   "name": "@enact/moonstone",
   "version": "4.5.0",
   "description": "Large-screen/TV support library for Enact, containing a variety of UI components.",
+  "repository": "https://github.com/enactjs/moonstone",
```

Comment:
Should it have `type: git` or not?

Reply:
Even without type:git, repository information is displayed properly.

Reply:
Would you please unify the format through the repos? If you want this, apply this format for all repos.

Reply:
add `type: git`